engine/config.h

```diff
@@ -0,0 +1,48 @@
+#ifndef _CONFIG_H_
+#define _CONFIG_H_
```

Comment:
I believe this is a build artifact too; it doesn't have the `generated by` lines in it but it's communicating what CMake found during the `cmake` operation.

Reply:
OK. Can we give this task to someone to do? There are a lot of these files and they really should be in gitignore.